### ISSUE TITLE
"Getting default value for OC not supported" erorr

### ISSUE THREAD

**Issue by dodyg** [mganss/XmlSchemaClassGenerator#45]
I am trying to generate these XSDs (http://www.iata.org/whatwedo/airline-distribution/ndc/Pages/schema-download2.aspx)

This command ```dotnet XmlSchemaClassGenerator.Console.dll edist_commontypes.xsd edist_structures.xsd```

generates

```
Unhandled Exception: System.NotSupportedException: Getting default value for OC not supported.
   at XmlSchemaClassGenerator.TypeModel.GetDefaultValueFor(String defaultString) in C:\projects\xmlschemaclassgenerator\XmlSchemaClassGenerator\TypeModel.cs:line 166
   at XmlSchemaClassGenerator.PropertyModel.AddMembersTo(CodeTypeDeclaration typeDeclaration, Boolean withDataBinding) in C:\projects\xmlschemaclassgenerator\XmlSchemaClassGenerator\TypeModel.cs:line 680
   at XmlSchemaClassGenerator.ClassModel.Generate() in C:\projects\xmlschemaclassgenerator\XmlSchemaClassGenerator\TypeModel.cs:line 349
   at XmlSchemaClassGenerator.NamespaceModel.Generate(String namespaceName, IEnumerable`1 parts) in C:\projects\xmlschemaclassgenerator\XmlSchemaClassGenerator\TypeModel.cs:line 54
   at System.Linq.Enumerable.SelectEnumerableIterator`2.MoveNext()
   at XmlSchemaClassGenerator.Generator.Generate(IEnumerable`1 files) in C:\projects\xmlschemaclassgenerator\XmlSchemaClassGenerator\Generator.cs:line 197
   at XmlSchemaClassGenerator.Console.Program.Main(String[] args) in C:\projects\xmlschemaclassgenerator\XmlSchemaClassGenerator.Console\Program.cs:line 134
```

**Comment by mganss** [comment]
Thanks for reporting this. I've implemented support for cases like these in 2.0.82-beta.

**Comment by mganss** [comment]
BTW, Here's what xsd.exe generates:
```C#
// CODEGEN Warning: DefaultValue attribute on members of type ServiceDefinitionTypeFeeMethod is not supported in this version of the .Net Framework.
// CODEGEN Warning: 'default' attribute supported only for primitive types.  Ignoring default='OC' attribute.
```
